### ISSUE TITLE
Error 577

### ISSUE THREAD

**Issue by Hasji** [Air14/HyperHide#31]
C:\Users\Rodj\Desktop\HyperHide\Scripts>sc start airhv
[SC] StartService: ошибка: 577:

Системе Windows не удается проверить цифровую подпись этого файла. При последнем изменении оборудования или программного обеспечения могла быть произведена установка неправильно подписанного или поврежденного файла либо вредоносной программы неизвестного происхождения.

C:\Users\Rodj\Desktop\HyperHide\Scripts>sc start HyperHideDrv
[SC] StartService: ошибка: 577:

Системе Windows не удается проверить цифровую подпись этого файла. При последнем изменении оборудования или программного обеспечения могла быть произведена установка неправильно подписанного или поврежденного файла либо вредоносной программы неизвестного происхождения.

**Comment by Air14** [comment]
Did you enable test sigining mode?

**Comment by Hasji** [comment]
Yep i`m enable sigining mode. But a not sigining and with sigining mode all sc 577

**Comment by Air14** [comment]
And what is your windows version?

**Comment by survivalizeed** [comment]
Having the same problem

**Comment by d2k2-git** [comment]
same problem. windows 10 22H2
